ckanext/nhsengland/plugin.py

```diff
@@ -119,6 +119,7 @@ def before_map(self, route_map):
         """
         with routes.mapper.SubMapper(route_map,
             controller='ckanext.nhsengland.plugin:NHSEController') as m:
+            m.connect('dashboard', '/dash', action='dashboard')
```

Comment:
Hey hey,

Regarding the recursive loop. It turns out that CKAN already has something at the end of the `/dashboard` path and by adding it (again) in our plugin weird things happen, birds fly backwards, the sun sets in the north and zombies attack. ;-)